src/Editor.js

```diff
@@ -1171,6 +1171,7 @@ const Editor = () => {
                     case "short-string":
                       return (
                         <div
+                          key={{key}}
```

Comment:
I believe this should be `key={key}` instead of `key={{key}}`.

Reply:
Thanks! Updated.